docroot/themes/contrib/civictheme/civictheme_library/.eslintrc.yml

```diff
@@ -8,6 +8,7 @@ globals:
   BACKGROUNDS: true
   ICONS: true
   LOGOS: true
+  SECONDARY_LOGOS: true
```

Comment:
could you please not do this and implement multiple logos discovery to be loaded into `LOGOS`  variable